src/main/java/uk/gov/companieshouse/officer/delta/processor/transformer/IdentityVerificationDetailsTransform.java

```diff
@@ -13,7 +13,6 @@ public class IdentityVerificationDetailsTransform implements Transformative<Delt
     private static final String IDENTITY_VERIFIED_ON = "identity_verified_on";
     private static final String APPOINTMENT_VERIFICATION_START_ON = "appointment_verification_start_on";
     private static final String APPOINTMENT_VERIFICATION_STATEMENT_DUE_ON = "appointment_verification_statement_due_on";
-    private static final String APPOINTMENT_VERIFICATION_STATEMENT_DATE = "appointment_verification_statement_date";
```

Comment:
Do we need the appointment_verification_statement_date field? I can see it is referenced in DeltaIdentityVerificationDetails.java.